README.md

```diff
@@ -1,31 +1,70 @@
 [RethinkDB](http://www.rethinkdb.com)
 =================
 
-RethinkDB is an open-source distributed database. It has an intuitive
-query language, automatically parallelized queries, and simple
-administration.
+RethinkDB is the first open-source scalable database built for realtime applications. It exposes a new database access model -- instead of polling for changes, the developer can tell the database to continuously push updated query results to applications in realtime. RethinkDB allows developers to build scalable realtime apps in a fraction of the time with less effort.
 
-To get started, check out [rethinkdb.com](http://rethinkdb.com)!
+To learn more, check out [rethinkdb.com](http://rethinkdb.com).
 
-Quick start
+Not sure what types of projects RethinkDB can help you build? Here are a few examples:
+
+* [Build a realtime liveblog with RethinkDB and Pubnub](http://rethinkdb.com/blog/rethinkdb-pubnub/)
```

Comment:
Could we make these links shorter? (not the phrases, just what's linked)

For example:

- Build a [realtime liveblog]() with RethinkDB and Pubnub